Javid_Alizada/05.Ansible_run/host_vars/deb01.yml

```diff
@@ -0,0 +1,2 @@
+ansible_host: 10.0.0.201
+ansible_ssh_private_key_file: /home/javid_alizada/DevOps_Traning/vagrant_project/ansible_box/.vagrant/machines/deb01/virtualbox/private_key
```

Comment:
could you please use vault for that

Reply:
Done. encrypt files in host_vars with vault.